paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandler.kt

```diff
@@ -84,7 +83,7 @@ internal class FlowControllerConfigurationHandler @Inject constructor(
             return
         }
 
-        AnalyticsRequestFactory.regenerateSessionId()
+        viewModel.resetSession()
 
```

Comment:
`FlowControllerConfigurationHandler` has a test that checks that the session ID is regenerated

Reply:
@tillh-stripe actual reseting of the session occurs here.